lib/matplotlib/lines.py

```diff
@@ -1031,6 +1050,25 @@ def set_color(self, color):
         self._color = color
         self.stale = True
 
+    def set_offcolor(self, offcolor):
+        """
+        Set the gap color for the dashed line style.
+
+        Parameters
+        ----------
+        offcolor : color or None
+            The gap color. If None, the gaps are unfilled.
+
+        """
+
+        if offcolor is not None:
+            if not is_color_like(offcolor) and offcolor != 'auto':
```

Comment:
Is there a reason to not use `mcolors._check_color_like(color=color)` as is done in `set_color`?

Reply:
Looks like `set_color` changed since the original PR.  I completely missed that and will change it.